src/main/java/com/codessquad/qna/web/user/UserController.java

```diff
@@ -0,0 +1,44 @@
+package com.codessquad.qna.web.user;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Controller
+@RequestMapping("/users")
+public class UserController {
+
+    private UserRepository userRepository = new UserRepository();
+
+    @RequestMapping(value = "/create", method = RequestMethod.POST)
+    public String user_create(User user) {
+        userRepository.add(user);
+        return "redirect:/users";
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public String user_list(Model model) {
+        model.addAttribute("users", userRepository.getUsers());
+        return "user/list";
+    }
+
+    @RequestMapping(value = "/{userId}", method = RequestMethod.GET)
+    public String user_profile(@PathVariable("userId") String userId, Model model) {
+        model.addAttribute("user", userRepository.getUser(userId));
+        return "user/profile";
+    }
+
+    @RequestMapping(value = "/{userId}/form", method = RequestMethod.GET)
+    public String user_form(@PathVariable("userId") String userId, Model model) {
+        model.addAttribute("user", userRepository.getUser(userId));
+        return "user/updateForm";
+    }
+
+    @RequestMapping(value = "/{userId}/update", method = RequestMethod.POST)
```

Comment:
```suggestion
    @RequestMapping(value = "/{userId}/update", method = RequestMethod.PUT)
```

으로 하고 싶은데, html 의 form 태그에서 
`<form name="question" method="put" action="/users/{{user.userId}}/update">`
처럼 메소드를 put 으로 지정할 수 있는 방법은 없을 까요?
form 의 메소드를 put 으로 지정하려하니, 제대로 요청이 가지 않아서요

Reply:
https://stackoverflow.com/questions/8054165/using-put-method-in-html-form

```html
<form method="post" ...>
  <input type="hidden" name="_method" value="put" />
...
```

Reply:
위의 stackoverflow 를 따라해보았는데 잘 되지 않네유 😭

Reply:
디온이 슬랙에 올렸던데 확인하셨죠?ㅋ